html/Axit/js/validation/contact-validation/contact-form-validation.js

```diff
@@ -0,0 +1,49 @@
+import {
+  checkName,
+  checkSubject,
+  checkMessage,
+  checkEmail,
+} from '../helper-functions.js';
+
+import {components} from './contact-components.js';
+
+export function addContactFormValidation() {
+  components.then((respose) => {
+    const {
+      contactForm,
+      errorMessages,
+      nameInput,
+      emailInput,
+      subjectInput,
+      messageTextArea,
+    } = respose;
+
+    contactForm.addEventListener('submit', () => {
+      event.preventDefault();
+      validateContactForm();
+    });
+
+    function validateContactForm() {
+      let result = [];
+
+      // Name validation
+      result.push(checkName(nameInput, errorMessages[0]));
```

Comment:
hardcoded values is not very good practice, what if you will have more elements? or they change places for some design reasons